spec/index.bs

```diff
@@ -3884,6 +3888,8 @@ shader module object, and {{Serializable}} means that the reference can be
 it concurrently. Since {{GPUShaderModule}} is immutable, there are no race
 conditions.
 
+Issue(gpuweb/gpuweb#354): Finish defining multithreading API and add `[Serializable]` back to the interface.
```

Comment:
Feels like we should either consistently add this issue to every interface we removed Serializable from so we can easily reference it later when adding them back or just put the issue in once for the entire file?

Reply:
I added it specifically in places where the text conflicts with the IDL, i.e. places where we describe serialization steps but no longer have `[Serializable]` on the interface. I think otherwise the presence of the GitHub issue is enough to track it.

Reply:
Ah, I see! Context was obscured by GitHub collapsing sections of the diff. LGTM, then!